src/Raven.Client/Http/RequestExecutor.cs

```diff
@@ -2365,6 +2369,7 @@ internal TestingStuff(RequestExecutor requestExecutor)
             internal int[] NodeSelectorFailures => _requestExecutor._nodeSelector.NodeSelectorFailures;
             internal ConcurrentDictionary<ServerNode, Lazy<NodeStatus>> FailedNodesTimers => _requestExecutor._failedNodesTimers;
             internal (int Index, ServerNode Node) PreferredNode => _requestExecutor._nodeSelector.GetPreferredNode();
+            internal Action DelayRequest => () => Thread.Sleep(18000);
```

Comment:
this is something that you need to set in the test, now everyone that will initialize testing stuff will have this delay

Reply:
Done